src/docs/common_tasks/multiple_jvm_versions.md

```diff
@@ -0,0 +1,53 @@
+
+## Problem
+
+You want to compile and/or run different test or binary targets using different JVMs.
+
+Or, you need to specify command-line options globally for the JVM on a per-JVM version basis. For
+example, using `--add-opens` flags, but only when running on JVMs that support it.
+
+## Solution
+
+Declare JVM platforms using the `jvm-platform` configuration section to define the set of available
```

Comment:
```suggestion
In your global configuration (most likely `pants.toml`), declare JVM platforms using the `jvm-platform` configuration section to define the set of available
```

I like the continuity of "in X1, do Y1. In X2, do Y2". In this case, X1 = global configuration and X2 = your target.